uwblueprint/community-fridge-kw
# fix: revert form values after cancelling edits

## Brief description. What is this change? 

### [Wrong value displayed after canceling change](https://www.notion.so/uwblueprintexecs/Wrong-value-displayed-after-canceling-change-109c929b58044ab5b74486d7429846f5)



## Implementation description. How did you make this change?
* Get request (`getScheduleById`) to get initial form values
* When cancel button is clicked, `discardChanges` is called - which reassigns all form values (for the corresponding page being edited) to their original values



## Steps to test
1. Schedule a donation - all values should be saved as usual
2. Edit the donation and click save - values in `Donation Details` page should update immediately in UI and persist after page refresh
3. Edit the donation and click cancel - values in `Donation Details` page should be the same as before (also after refresh)
4. Test Steps 2 & 3 for `Drop-off Information`, `Donation Information` and `Volunteer Information`




## Checklist
